src/lib/impl/SubscriberImpl.js

```diff
@@ -131,10 +133,12 @@ class SubscriberImpl extends EventEmitter {
     this._log.debug('Shutting down subscriber %s', this.getTopic());
 
     Object.keys(this._pubClients).forEach(this._disconnectClient.bind(this));
+    Object.keys(this._pendingPubClients).forEach(this._disconnectClient.bind(this));
```

Comment:
This won't really do anything though, because the clients won't be in _pubClients though, right?